release_notes/ocp-4-11-release-notes.adoc

```diff
@@ -3182,4 +3182,23 @@ $ oc adm release info 4.11.24 --pullspecs
 [id="ocp-4-11-24-updating"]
 ==== Updating
 
-To update an existing {product-title} 4.11 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc[Updating a cluster using the CLI] for instructions. 
+To update an existing {product-title} 4.11 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc[Updating a cluster using the CLI] for instructions.
+
+[id="ocp-4-11-25"]
+=== RHSA-2023:0245 - {product-title} 4.11.25 bug fix and security update
+
+Issued: 2023-01-23
+
+{product-title} release 4.11.25, which includes security updates, is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHSA-2023:0069[RHSA-2023:0245] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHBA-2023:0068[RHBA-2023:0244] advisory.
```

Comment:
These advisories link to 4.11.24. Link sends to 0069, but text in doc says 0425. Maybe I'm missing something?